### DISCUSSION TITLE
Canva Pro option is not showing up

### DISCUSSION THREAD

**Discussion by Chiruu1** [community/community#29023] [GitHub Education]
Hi, I have created a new account using my college id but the Canva Pro option is not showing up in the "Virtual event kit" because of which I'm unable to get Canva Pro subscription.
![image](https://user-images.githubusercontent.com/110601270/183913467-6b726da5-0c86-42b3-ad23-dc797ae988fd.png)
#28530

**Comment by mlarasusan** [comment]
The Canva benefit is no longer offered in the GitHub Student Developer Pack.
For students who are already on the 12 month benefit, you will continue to have access for the remainder of your 12 month offer period & can contact Canva's support team with any related questions.

**Reply by staimur121** [reply]
Then how to apply for Canva pro student pack with out github

**Comment by nuralamsyah03** [comment]
canva tidak lagi bekerjasama dengan Github pro

**Reply by dwijohdi** [reply]
pantes akun canva tiba tiba ga pro lagi, jd bener2 ga bisa akses yang pro lagi ya?
